mlflow/models/evaluation/default_evaluator.py

```diff
@@ -669,9 +659,14 @@ def _log_model_explainability(self):
             )
             _logger.debug("", exc_info=True)
 
+        def _adjust_color_bar():
+            pyplot.gcf().axes[-1].set_aspect('auto')
+            pyplot.gcf().axes[-1].set_box_aspect(50)
```

Comment:
This fixed color bar renderring:

Before:
![shap_summary_plot_on_data_iris_pandas_df_longname_cols_dataset](https://user-images.githubusercontent.com/19235986/188812695-cd8c2e32-a56b-4d03-a4bc-81c94cdc97ce.png)

After:
![shap_summary_plot_on_data_iris_pandas_df_longname_cols_dataset](https://user-images.githubusercontent.com/19235986/188812733-e7f1ea1b-2835-4f2c-b99e-654d2e278d25.png)